pkg/assembler/graphql/schema/isDependency.graphql

```diff
@@ -44,6 +44,8 @@ type IsDependency {
   origin: String!
   "GUAC collector for the document"
   collector: String!
+  "Reference location of the document in the persistent blob store (if that is configured)"
+  documentRef: String!
```

Comment:
I think these should be `String` (and use `null` when location is not configured). Or do we want to special case `""` in code instead and require this all the time?

Reply:
For all the "verbs" or just the `isDependency` or `isOccur` ones?

Reply:
All verbs, I think we should be consistent

Reply:
in that case the `inputSpec` would also have to be just `String` for `documentRef`

Reply:
Yes. All will be optional

Reply:
hmm all the other `inputSpec` we require. I rather keep it consistent and make it an empty string `""`. What do you think?

Reply:
But then you would have to always create and pass an empty string in code to build the request and then on resolver side have a special check for `""` (we don't want to store empty strings in the db if we can avoid it, in general). For `String`, you only need to perform the check against `nil` in resolver, but client doesn't need to change if there is no document.

Reply:
Overall, I'm ok either way, but we should still have a discussion if we want to take the cost of specifying an additional parameter each time or not

Reply:
Makes sense, I can make it optional.

Reply:
Updated to make it optional `String` for both the `type` and `InputSpec`